src/app/GUI.py

```diff
@@ -0,0 +1,82 @@
+import sys
+from dataclasses import dataclass
+
+import pygame
+import time
+from src.app._app import App
+from src.domain.zemlja import Zemlja
+from src.app.GUItext import messages
+
+
+@dataclass
+class GUI(App):
+    width: int
+    height: int
+    zemlja = None
+    tocke: int = 0
+    dx: int = 0
+    dy: int = 0
+
+    def __post_init__(self):
+        pygame.init()
+        self.windowSurface = pygame.display.set_mode((self.width, self.height), 0, 32)
+
+    def draw_game(self):
+        pygame.display.set_caption("Snake Game")
+        # zapolni display z barvo, bela
+        self.windowSurface.fill((255, 255, 100))
+        self.dx = self.width / 20
+        self.dy = self.height / 20
+        # nariši kačo
+        pygame.draw.rect(self.windowSurface, (0, 0, 0), (
```

Comment:
Tukaj bi bilo zelo fino dodati imena argumentov na tak način da je bolj pregledno in bolj varno če boš v prihodnosti delal kakšne spremembe ali pa če bo pygame v prihodnosti zamenjal vrstni red argumetov...
```
def test(a: int):
    ...

test(a=10)
```
Ker ne poznam na pamet pygame nimam pojma kaj pomenijo kaki argumenti pri tem klicu...
```
ygame.draw.rect(
      self.windowSurface,
      (0, 0, 0),
      (self.zemlja.snake.x * self.dx, self.zemlja.snake.y * self.dy,
      (20),
      (20)))
```
Če pogledava dokumentacijo... https://www.pygame.org/docs/ref/draw.html#pygame.draw.rect
Lahko potem tole napiševa takole...
```
ygame.draw.rect(
      surface=self.windowSurface,
      color=(0, 0, 0),
      rect=(
            self.zemlja.snake.x * self.dx, self.zemlja.snake.y * self.dy,
           (20),
           (20)
      )
)
```